mokapot/model.py

```diff
@@ -17,6 +17,8 @@
 import logging
 import pickle
 import warnings
+import math
```

Comment:
Is this used anywhere?

Reply:
Nope! Had it in for testing, forgot to remove